node/core/backing/src/lib.rs

```diff
@@ -528,7 +528,9 @@ impl CandidateBackingJob {
 								descriptor: candidate.descriptor.clone(),
 								commitments,
 							});
-							self.sign_import_and_distribute_statement(statement, parent_span).await?;
+							if let Some(stmt) = self.sign_import_and_distribute_statement(statement, parent_span).await? {
```

Comment:
It looks like this code is wider than [120 maximum](https://github.com/paritytech/polkadot/wiki/Style-Guide). Can we wrap this code?